src/components/Personalization/createViewCollect.js

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { isNonEmptyArray, assign } from "../../utils";
+
+export default ({ eventManager, mergeMeta }) => {
+  return ({ meta, xdm = {} }) => {
+    const { decisions = [] } = meta;
+    const data = { eventType: "display" };
```

Comment:
Do we need a more descriptive event type here? `display` can mean anything.

Reply:
Or is it because we are sending this information in `data`, only Target cares about this? Which I don't understand why we are not logging this as a regular experience event.

Reply:
this is not something that has been introduced recently. The notifications for `__view__` are already sent as `xdm.eventType: "display"` and it was discussed in the wiki a long time ago.